sdk/common/src/main/java/io/opentelemetry/sdk/resources/ResourcesConfig.java

```diff
@@ -103,8 +105,11 @@ protected Builder fromConfigMap(
       String stringValue = getStringProperty(OTEL_JAVA_DISABLED_RESOURCES_PROVIDERS, configMap);
       if (stringValue != null) {
         this.setDisabledResourceProviders(
-            ImmutableSet.copyOf(
-                Splitter.on(',').omitEmptyStrings().trimResults().split(stringValue)));
+            Collections.unmodifiableSet(
+                Arrays.stream(stringValue.split(","))
+                    .filter(s -> !s.isEmpty())
+                    .map(String::trim)
```

Comment:
I think you might need to swap the order of trim/isEmpty to match Guava. AFAIK `omitEmptyStrings` only sets a boolean. But I may be wrong here.

Reply:
Whether or not it matches guava, this logic seems right. Values that are only whitespace shouldn't be used, I don't think.

Reply:
My not splitting the PRs up in advance caused this review to be a bit hectic, sorry about that. While GitHub doesn't have the outdated mark here, if I remember correctly, the code used to be in the wrong order and @Oberon00 commented and I fixed it so it should be good now.